web/src/OverviewTable.tsx

```diff
@@ -539,21 +551,63 @@ export function ResourceTableHeadRow({
   )
 }
 
+function ShowMoreResourcesRow({
+  colSpan,
+  itemCount,
+  pageSize,
+  onClick,
+}: {
+  colSpan: number
+  itemCount: number
+  pageSize: number
+  onClick: (e: MouseEvent) => void
+}) {
+  if (itemCount <= pageSize) {
+    return null
+  }
+
+  return (
+    <ResourceTableRow style={{ height: "45px" }}>
```

Comment:
i'd slightly prefer we have something like 

```
&.isNarrow {
  height: 45px
}
```

in the styled component, so it's a bit easier to see all the style combinations in one place

Reply:
good point!